.github/actions/validate-openapi-spec/action.yml

```diff
@@ -0,0 +1,30 @@
+name: validate-openapi-spec
+description: Validate OpenAPI specification compliance
+runs:
+  using: composite
+  steps:
+    - name: Install poetry
+      shell: bash
+      run: pipx install poetry
+
+    - name: Set up Python
+      uses: actions/setup-python@v5
```

Comment:
Directly use `v6` here, as there's a pending dependabot PR to update it in other places (#1410).

Reply:
Updated